MobileCenter/MobileCenter/Internals/Utils/MSUtility+File.h

```diff
@@ -0,0 +1,36 @@
+#import <Foundation/Foundation.h>
+#import "MSUtility.h"
+
+/*
+ * Workaround for exporting symbols from category object files.
+ */
+extern NSString *MSUtilityFileCategory;
+
+/**
+ * Utility class that is used throughout the SDK.
+ * File part.
+ */
+@interface MSUtility (File)
+
+/**
+ * Creates a file at the given location, intermediate directories are also create if nonexistant.
+ *
+ * @param fileURL URL representing the absolute path of the file to create.
+ *
+ * @return `YES` if the operation was successful or if the item already exists, otherwise `NO`.
+ *
+ * @discussion SDK files should not be backed up in iCloud. Thus, iCloud backup is explicitely
+ * deactivated on every folder created.
+ */
++ (BOOL)createFileAtURL:(NSURL *)fileURL;
+
+/**
+ * Removes the file or directory at the specified URL.
+ *
+ * @param itemURL URL representing the absolute path of the file to create.
+ *
+ * @return YES if the item was removed successfully or if URL was nil. Returns NO if an error occurred.
+ */
++ (BOOL)removeItemAtURL:(NSURL *)itemURL;
```

Comment:
`removeItemAtURL` is also the Apple naming for it so I suggest to keep it like this.